sig-cf/README.md

```diff
@@ -0,0 +1,29 @@
+# SIG: CF
+
+This SIG focuses on the current Cloud Foundry (CF) clouddriver provider for Spinnaker.
+
+## Meetings
+
+* Regular SIG Meeting
+  * Last Thursday of each month, at 10 AM US/Pacific ([See in your time zone](https://www.thetimezoneconverter.com/?t=10am&tz=San%20Francisco))
+  
+  [Agenda](https://docs.google.com/document/d/1TouFx5ykNCcZxNSD2JhUiRliZDXwxZ6sx239rigYdxw/edit?usp=sharing)
+
+## Leadership
+
+* Zach Smith - Armory (https://github.com/zachsmith1)
+* Ryan Pei - Armory (https://github.com/ryanpei)
+
+## Contact
+
+* [Slack](https://spinnakerteam.slack.com/archives/C012PGYN771)
+
+## Scope
+
+*Status and Alignment*
+  * [CF Clouddriver](https://github.com/spinnaker/clouddriver/tree/master/clouddriver-cloudfoundry)
```

Comment:
This appears to be narrowly scoped to just cloudfoundry in `clouddriver` ... I assume this SIG will also cover CF-related issues that may pop up in the UI and orchestration tiers?

Reply:
Yes, is this more clear?
```suggestion
  * Issues in [CF Clouddriver](https://github.com/spinnaker/clouddriver/tree/master/clouddriver-cloudfoundry) and other Spinnaker services which affect deployments to CloudFoundry
```